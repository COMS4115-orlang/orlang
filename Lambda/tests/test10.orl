val operator_if : Bool -> 'aaa -> 'aaa -> 'bbb
let operator_if c t e = _if c _then t _else e

let random x y =
  match x with
  | 5 = [3]
  | 3 = [4, 5, 6];

