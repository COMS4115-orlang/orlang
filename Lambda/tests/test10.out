Fatal error: exception Failure("Semantic checking for pattern matching not implemented yet")
